src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -9,41 +9,107 @@
 -->
 
 <template>
-  <div class="quick-navigation-modal">
-    <div class="container">
+  <div
+    class="quick-navigation"
+    @keydown.down.prevent="handleKeyDown"
+    @keydown.esc.prevent="closeQuickNavigationModal"
+    @keydown.enter="handleKeyEnter"
+  >
+    <div
+      class="quick-navigation__modal-shadow"
+      @click="closeQuickNavigationModal()"
+    >
+    </div>
+    <div
+      class="quick-navigation__container"
+    >
+      <div class="quick-navigation__input-container">
+        <div class="quick-navigation__magnifier-icon-container">
+          <MagnifierIcon />
+        </div>
+        <input
```

Comment:
How is that we aren't using FilterInput here? We could reuse most of the HTML and CSS code so this component would become much smaller.

Reply:
Refactor made in https://github.com/apple/swift-docc-render/pull/396/commits/eb4841068d99750abcfe827c5935efe2862901a8, thx for the suggestion!